app/components/groups/group-list/group-list.controller.js

```diff
@@ -4,8 +4,29 @@
 
     .controller("GroupListController", GroupListController)
 
-    GroupListController.$inject = ["$scope"]
+    GroupListController.$inject = ["$scope", "GroupResource"]
+
+    function GroupListController($scope, GroupResource){
+
+      getGroups(0)
+
+      $scope.paginatorConfig = {
+        currentPage: 1,
+        itemsPerPage: 10,
+        changePage: function(){
+          var currentPage = $scope.paginatorConfig.currentPage - 1
+          var offset = currentPage * $scope.paginatorConfig.itemsPerPage
+          getGroups(offset)
+        }
+      }
+
+      function getGroups(offset){
+        GroupResource.getGroups({offset: offset}).$promise
```

Comment:
it's better to include `limit` in API calls also since it is a common feature to change number of items for the page